schemas/2018-02-02/Microsoft.Migrate.json

```diff
@@ -0,0 +1,607 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2018-02-02/Microsoft.Migrate.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Migrate",
+  "description": "Microsoft Migrate Resource Types",
+  "resourceDefinitions": {
+    "projects": {
```

Comment:
>projects [](start = 5, length = 8)

please add the new resource types to schemas/2014-04-01-preview/deploymentTemplate.json and schemas/2015-01-01/deploymentTemplate.json.

Reply:
Done. Thanks.